mycli/main.py

```diff
@@ -832,7 +856,23 @@ def format_output(title, cur, headers, status, table_format, expanded=False, max
         headers = [utf8tounicode(x) for x in headers]
         if expanded:
             output.append(expanded_table(cur, headers))
-        elif table_format is not None:
+        elif table_format is None:
+            output.append('\t'.join(headers))
+            for row in cur:
+                output.append('\t'.join([str(r) for r in row]))
```

Comment:
I think you can use our tabulate library for this. It has a `tsv` format. It looks like if you were to set `table_format` to `'tsv'` if it is `None`, then the `else:` statement below will take care of this code.